openedx/adg/lms/applications/models.py

```diff
@@ -83,6 +84,13 @@ def submit_application_for_current_date(self):
         self.submission_date = date.today()
         self.save()
 
+    @property
+    def is_written_application_started(self):
+        try:
+            return bool(self.user.application)
+        except ObjectDoesNotExist:
+            return False
```

Comment:
Maybe a better way can be 
```suggestion
        return hasattr(self.user, 'application')
```

Reply:
These are'not simple python classes. hasattr deals with AttributeError but not with ObjectDoesNotExist exception. getattr also make use of hasattr.

Reply:
I have tried hasattr and getattr and they both work fine in our case. Lets try these If they don't work we can skip but it worked for me.

Reply:
Yes it works only when user has started application but won't work in all cases. like if user doesn't have related application it will raise RelatedObjectDoesNotExist exception which isn't handled in suggested method.